<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QDialog" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>443</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>443</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1230</width>
    <height>443</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Депозитный калькулятор</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_deposit_sum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>127</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Сумма вклада</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_term">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Срок размещения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent_rate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_select_periodicity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Периодичность выплат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_withdrawal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Частичные снятия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_refill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Частичные пополнения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_capitalization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_deposit_sum">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deposit_sum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deposit_term">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percent_rate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent_rate_unit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_capitalization">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_deposit_refill">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_deposit_withdrawal">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deposit_refill">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deposit_withdrawal">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_periodicity">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="select_refill">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="select_withdrawal">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_tax_rate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Налоговая ставка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tax_rate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>13.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tax_rate_unit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_refill">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_withdrawal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_term_unit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>мес.</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_percent_accrued">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nalog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Налог</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_sum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Сумма на вкладе к концу срока</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_percent_accrued">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_nalog">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currency_total_deposit_sum">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="percent_accrued">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="nalog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_sum_end_term">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
qproperty-alignment: 'AlignVCenter | AlignRight';</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_calculate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>610</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
